src/Tooling/Bit.Tooling.Templates/TodoTemplate/Web/Pages/EditProfile.razor.cs

```diff
@@ -6,15 +6,19 @@ public partial class EditProfile
 {
     public UserDto? User { get; set; } = new();
 
+    public string? FullName { get; set; }
+    public DateTimeOffset? BirthDate { get; set; }
     public string? SelectedGender { get; set; }
```

Comment:
We want to enable or disable the save button based on if the user input was different from what was in the database.
So we have to not change the value stored in UserDto and put user inputs in new variables so that it is finally comparable.